docker/funsize-update-generator/scripts/funsize.py

```diff
@@ -224,9 +229,7 @@ def main():
         # if branch not set explicitly use repo-name
         mar_data["branch"] = e.get("branch",
                                    mar_data["repo"].rstrip("/").split("/")[-1])
-        mar_name = "%(appName)s-%(branch)s-%(version)s-%(platform)s-" \
-                   "%(locale)s-%(from_buildid)s-%(to_buildid)s.partial.mar" % \
-                   mar_data
+        mar_name = args.filename_template.format(**mar_data)
```

Comment:
ah, right, so we can call format() on this even if it's the explicit filename (no interpolation) sent from releasetasks